scss/main.scss

```diff
@@ -125,8 +124,7 @@ h1,h2,h3 {
   display: flex;
   background-color: $cfa-red;
   align-items: center;
-  justify-content: center;
-
+  padding-left: 6rem;
```

Comment:
Can you revisit this change?
It's currently misaligning the hero on mobile:

<img width="420" alt="Screen Shot 2021-02-18 at 8 46 28 AM" src="https://user-images.githubusercontent.com/9326234/108373714-cefb3e00-71c5-11eb-97f1-9cd89afefa07.png">

Reply:
Ooh yes I removed the padding at the breakpoint and it looks like it's behaving as expected now.